### PR TITLE
Bump version to 0.3.0

### PR DESCRIPTION
**Pull request by aleksul**
Follows PRs #46 #60, depends on PRs #66 #67 #68.

I think we should release a minor update, because a lot of improvements have been made since last release.

Also, I've fixed `Headings should be surrounded by blank lines` rule in `CHANGES.md`.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # Change log
 
+## v0.2.1
+
+- Add Windows and Debian support. Add Ubuntu 16.04 and 20.04 support as well.
```
Comment (AustinScola) [inline CHANGES.md]:
```suggestion
- Add support for Windows, Debian, Ubuntu 16.04, and Ubuntu 20.04.
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+- Add separate class for mongod binary.
+- Separate unit and integration tests.
+- Combined all the configs in pyproject.toml.
```
Comment (AustinScola) [inline CHANGES.md]: Do you think these belong in the change log? I'm unsure about if internal changes should be in a change log or how much they should be explained?

File: CHANGES.md [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+- Add "how it works" and "installation" sections to readme.
```
Comment (AustinScola) [inline CHANGES.md]:
```suggestion
- Improve the readme.
  - Add section about how `pytest-motor` works.
  - Add section about installation.
```

**Comment by AustinScola** [comment]
This should probably be `0.3.0` not `0.2.1` because adding support for multiple systems is a minor change, not a patch or bugfix.

**Comment by AustinScola** [comment]
> Also, I've fixed `Headings should be surrounded by blank lines` rule in `CHANGES.md`.

Where is this rule coming from?

**Comment by aleksul** [comment]
> 
> 
> > Also, I've fixed `Headings should be surrounded by blank lines` rule in `CHANGES.md`.
> 
> Where is this rule coming from?

There is a [markdown linter](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022) (and a plugin for VS Code powered by it, which I'm personally using) and I think it's pretty cool.

File: CHANGES.md (hunk 2 above)
Comment (aleksul) [reply CHANGES.md]: I'm not sure either :) Maybe remove lines 7-8, replace line 6 with something like "Improved mongod binary handling"?

File: CHANGES.md [hunk 4]
```diff
@@ -1,6 +1,15 @@
 # Change log
 
+## v0.3.0
+
+- Add support for Windows, Debian, Ubuntu 16.04, and Ubuntu 20.04.
+- Improve mongod binary handling.
+- Improve the readme.
```
Comment (aleksul) [inline CHANGES.md]: Prettified

**Comment by AustinScola** [comment]
> > > Also, I've fixed `Headings should be surrounded by blank lines` rule in `CHANGES.md`.
> > 
> > 
> > Where is this rule coming from?
> 
> There is a [markdown linter](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022) (and a plugin for VS Code powered by it, which I'm personally using) and I think it's pretty cool.

Okay, I actually kind of disagree with this rule. I think it is better to have the text close to the header that it is associated with. This is a design principle I learned from the book _The Non-Designer's Design Book_:
> The Principle of Proximity states: Group related items together. Move them physically close to each other so the related items are seen as one cohesive group rather than a bunch of unrelated bits.

**Comment by aleksul** [comment]
> Okay, I actually kind of disagree with this rule. I think it is better to have the text close to the header that it is associated with. This is a design principle I learned from the book _The Non-Designer's Design Book_:
> 
> > The Principle of Proximity states: Group related items together. Move them physically close to each other so the related items are seen as one cohesive group rather than a bunch of unrelated bits.

That's a good point! Both variants look the same after formatting anyway.

**Pull request merged**